### PR TITLE
yonghao's first post

### PR DESCRIPTION
**Pull request by yonghaoy**
I post a blog which is about UNC's basketball game news. 
Who can tell me, did duke win??? The blog knows! 
I did a lot job on the blog, it looks cool, I like the unc blue background!

http://yonghaoy.github.io/spring2014/2014/02/22/yonghao%27s-blog.html

### CODE REVIEW

File: _posts/2014-02-22-yonghao's-blog.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+author: yonghaoy
```
Comment (eah13) [inline _posts/2014-02-22-yonghao's-blog.md]: Your author name isn't showing up in the post.  I suspect an error here.

**Comment by eah13** [comment]
@yonghaoy Great job.  You're one step ahead of us, making your route, which is great.  Fix that author name and you're ready for a merge.

**Comment by gerbal** [comment]
fixes #314

**Comment by yonghaoy** [comment]
my about page: https://github.com/silshack/spring2014/issues/316
http://yonghaoy.github.io/spring2014/2014/02/28/yonghaosecondblog.html

**Comment by eah13** [comment]
Nice work!  Sorry for the slow review time

**Pull request merged**